sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -500,6 +502,17 @@ public OutT apply(KafkaRecord<KeyT, ValueT> record) {
             return new KafkaConsumer<>(config);
           }
         };
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+      builder.add(DisplayData.item("topics", Joiner.on(",").join(getTopics())));
+      for (Map.Entry<String, Object> conf : getConsumerConfig().entrySet()) {
```

Comment:
Agree with @rangadi, this is not `LOG.DEBUG`, I'd go through Kafka's documentation and add here the "important configurations".
Maybe pick the once marked as "high importance" in [Kafka documentation](https://kafka.apache.org/documentation/#consumerconfigs).